package.json

```diff
@@ -85,7 +85,9 @@
     "rtl-detect": "^1.0.0",
     "sweetalert": "^2.1.0",
     "sweetalert-react": "^0.4.11",
+    "tmp": "0.0.33",
     "transform-runtime": "0.0.0",
+    "wacs-client": "0.1.0",
```

Comment:
Only if that's a requirement.  WA asked me to make the client, so I named it WACS, even though it implements the others too.  Note that this nomenclature doesn't leak into anything user-visible.

Reply:
...but we're not proud and will rename this if desired.  Another option would be for Autographa to absorb this code and kill the module.  It's very tiny, and although it'd be great to expand it in the future, it's really only applicable to Autographa at this stage.  It's almost silly for it to be its own module.

Reply:
That's fair. I think since it is only internal for now, we could go ahead with it until we either put the code directly in the code base or the module is refactored in the future at some point.